examples/save-to-disk/main.go

```diff
@@ -98,6 +101,21 @@ func main() {
 	// This will notify you when the peer has connected/disconnected
 	peerConnection.OnICEConnectionStateChange(func(connectionState webrtc.ICEConnectionState) {
 		fmt.Printf("Connection State has changed %s \n", connectionState.String())
+
+		if connectionState == webrtc.ICEConnectionStateConnected {
+			fmt.Println("Ctrl+C the client to stop the demo")
```

Comment:
This means `.Close()` will never get called right? If you want to support `Ctrl-C` that would be really cool also?

I would put a note in the `README` the created `ivf and opus files will exist, but headers will only have temporary values and may not play properly` if they don't end the example properly.

Reply:
Oh this needs to be more explicit then. By client, I meant the JS fiddle thing for example. If you "ctrl+c" it (or close the tab), the Go program will detect the connection is closed, will update the headers, close the files and terminate.

I thought about supporting ctrl+c in the Go example, but also wanted to keep the example simple. I can add it if you think it's relevant!